Incident Wash-Up

```diff
@@ -0,0 +1,19 @@
+Incident Wash-Ups should be used to discuss issues that have affected our customers and how we can help to avoid similar situations from occurring in the future. The Wash-Up should include all relevant parties at their earliest opportunity after the event. 
+
+Please use this format for all Incident Wash-Ups so that we have a consistent set of data on how we handle situations. Please add all notes from Incident Wash-Ups to the Confluence space (https://holidayextras.jira.com/wiki/display/WEB/Incident+Wash-Ups), for reference.
+
+    1. Summary
+    
+    2. Affected time period
+    
+    3. Parties involved
+    
+    4. Factors that have contributed 
```

Comment:
If I were running this meeting I would take it to mean "what caused the problem" so could be:
no tests so something went live broken or AWS went down or other third party failure or a combination of all those things. Perhaps wording such as "what was the cause" would suit better?